libs/cases/backend/case-management/src/lib/services/case.service.ts

```diff
@@ -23,12 +30,15 @@ export class XdCaseService {
 	 */
 	public getAllCases(): Observable<ICaseResponse[]> {
 		return from(this.prismaService.case.findMany()).pipe(
-			map((items) =>
-				items.map((item) => ({
+			map((items) => {
+				if (items.length === 0) {
+					throw new NotFoundException('No Cases found.');
```

Comment:
Why should this be an exception? When no case has been created yet, there should Error be thrown